.github/workflows/react-native.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       run: yarn install
 
     - name: Build
-      run: yarn bootstrap
+      run: yarn
```

Comment:
can we remove test? We don't have any unit tests for rn, so it seems silly to have it